NeverGreenGrove/NeverGreenGrove/src/Texture.h

```diff
@@ -1,31 +1,51 @@
-#ifndef TEXTURE_H
-#define TEXTURE_H
+/*
+MIT License
 
-#pragma once
+Copyright (c) 2017   Chris McArthur, prince.chrismc(at)gmail(dot)com
+                     Daniel P, privorotskyd(at)gmail(dot)com
+                     Nicholas G, dj_nick_gattuso(at)hotmail(dot)com
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
 
-#include "gl\glew.h"
+#pragma once
 
 class Texture
 {
 public:
-	Texture() {}
-	void createTexture(const char* imagePath);
-	float getCoordinates();
-	unsigned int getTexture();
+   Texture() {}
+   void createTexture(const char* image_path);
```

Comment:
I think this should be private. you don't want to be able to change the loaded texture during the application.
make the ctor take a `const char *` and in the body call `CreateTexture(...)`

Reply:
I can't say that I 100% understand what you mean...but I'll take a look at it